tests/search_tests.ts

```diff
@@ -48,7 +48,7 @@ const dataset = [
   },
   {
     id: 4,
-    title: 'Harry Potter and the Half-Blood Prince',
+    title: "Harry Potter and the Half-Blood Prince",
```

Comment:
Why do you remove the single quotes?

Reply:
Because of our private conversation .. I forgot to put them back